src/main/java/com/ingress/ms/subscription/dao/repository/SubscriptionRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.ingress.ms.subscription.dao.repository;
+
+import com.ingress.ms.subscription.dao.entity.SubscriptionEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface SubscriptionRepository extends JpaRepository<SubscriptionEntity, Long> {
```

Comment:
1. [Optional] - You can use CrudRepository 2. Change method implementation to jpa